### ISSUE TITLE
Providing Specimens 

### ISSUE THREAD

**Issue by nguyenvivi** [GrantThorntonETSI/eFile_Reimagined#389]
Scenario 1: 1(a) and 1(b) filing bases.  
User selects "No, the mark is not in use with all of the selected goods/services."

![image](https://user-images.githubusercontent.com/46028434/53896163-75fa6580-4001-11e9-8248-7920b82fbd77.png)

**Comment by l-stanny** [comment]
@serra-trinh @JackieBabos completed locally, pushed to GitHub

**Comment by dutchTender** [comment]
this is turning out to be even more difficult then i imagined.
i thought we could call the click on the goods and services level and trigger the save options and reveal and hide logic for the goods and services
but that is not the case, when user click at the class level
i can not just call the buttons on the goods and services level and call click on it
and have it trigger auto save and hide and reveal the sections it controls
i basically will need to write out the logic again when the calls are made from the class level
instead from the goods and services level

you can see some of this behavior in the static file as well, but in different context
select yes at the class level and then select some yes at the goods and services level
then hit no at the class level
and toggle
you will see that only the radios are being affected, the area that each radio button control is not affected ...but the bigger problem here is the auto save is not being called
so basically the logic that are on each radio button at the goods and services level, will need to be rewritten when they are being called as a group when user select no or yes at the class level

**Comment by dutchTender** [comment]
![image](https://user-images.githubusercontent.com/7011252/55255156-4ae5e900-5230-11e9-81f5-a105548c5676.png)

**Comment by nguyenvivi** [comment]
Let's discuss this at the eFile mid-week checkpoint meeting tomorrow.

**Comment by JackieBabos** [comment]
When the User selects "One specimen for all goods/services within this class", then within each individual good or service, the checkbox "Check here to provide a specimen for this individual good/service" should be removed. Since the user uploaded the specimen at the class level. 
![image](https://user-images.githubusercontent.com/44168600/55505088-907b2b00-560f-11e9-9b32-24c5d3f2b1a2.png)
![image](https://user-images.githubusercontent.com/44168600/55505116-a25cce00-560f-11e9-9dac-9dfa34e37666.png)

**Comment by JackieBabos** [comment]
Split this issue, the Specimen checkboxes are now Issue #452.
